extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduler.java

```diff
@@ -18,4 +18,9 @@
      */
     void resume();
 
+    /**
+     * @return if the scheduler is running
```

Comment:
I think that we should be more specific here. Something like "If a scheduler is running the triggers are fired and jobs are executed."